src/AbstractFixer.php

```diff
@@ -97,6 +91,13 @@ public function getDefinition()
         );
     }
 
+    /**
+     * Legacy, will be removed if proper description has been added to all fixers.
+     *
+     * @return string
+     */
+    abstract protected function getDescription();
```

Comment:
>not needed, there is already getDefinition in this AbstractFixer.

I removed that one ;)

>With this, you require the concrete fixer to implement getDescription method even if getDefinition will return full definition

by the end of the PR I'll make sure there won't be missing descriptions for our fixers and this can be removed here altogether